packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx

```diff
@@ -26,7 +26,7 @@ import type { ComboboxProps, ComboboxState } from './Combobox.types';
  * @param ref - reference to root HTMLElement of Combobox
  */
 export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLInputElement>): ComboboxState => {
-  const baseState = useComboboxBaseState(props);
+  const baseState = useComboboxBaseState(props, true);
```

Comment:
Nit: It can be nice to take an options argument that is an object so that callers can see what true means.